src/winter_onboarding_2021/fleet_management_service/routes.clj

```diff
@@ -37,9 +39,13 @@
         ["users" {"/signup" {:get (wrap-layout user-handlers/signup-form)
                              :post user-handlers/create-user}
                   "/login" {:get (wrap-layout user-handlers/login-form)
-                            :post user-handlers/login}}]
+                            :post user-handlers/login}
+                  "/dashboard" {:get (authentication-required (wrap-layout dashboard-handlers/index)
+                                                              #{:admin :manager :driver})}}]
```

Comment:
To allow `any` authenticated user, we are now passing all users in the set. Could this behaviour be changed, 

Like a route, which requires user to be logged-in, irrespective of the role.